cli/runner/runner.go

```diff
@@ -4,13 +4,16 @@ import (
 	"context"
 	"github.com/saucelabs/saucectl/cli/command"
 	"github.com/saucelabs/saucectl/cli/config"
+	"github.com/saucelabs/saucectl/cli/utils"
 	"github.com/saucelabs/saucectl/internal/fleet"
 )
 
 const logDir = "/var/log/cont"
 
+var homeDir, _ = utils.GetHomeDir()
```

Comment:
@dpgraham do we want to handle the error? or warn about it?

Reply:
@vrunoa I changed it to a warning and made it so GetProjectDir defaults to `/home/seluser` if it can't get working directory.